L/libmxml/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libmxml"
+version = v"4.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/michaelrsweet/mxml.git", "0d5afc4278d7a336d554602b951c2979c3f8f296")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd mxml/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make 
+make install
+chmod 666 /workspace/destdir/lib/pkgconfig/mxml4.pc
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("riscv64", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; )
+]
```

Comment:
```suggestion
platforms = supported_platforms()
filter!(p -> (Sys.islinux(p) || Sys.isapple(p)), platforms)
```
You only seem to be building for Linux and Apple, so this is the cleaner way of representing this.

Also, what would be the issues with building on the other platforms? Does upstream not support them?

Reply:
Fair point. It's my first JLL and I wasn't thinking much about other platforms I'm not using. 
Let me fix that.